### ISSUE TITLE
next steps for (plausible) analytics on the website

### ISSUE THREAD

**Issue by pietroppeter** [nim-lang/website#342]
with https://github.com/nim-lang/website/pull/339 we added plausible analytics to the website which are now publicly visible here: https://plausible.io/nim-lang.org

as a follow up on that I see the following steps:

- [ ] [import google analytics](https://plausible.io/docs/google-analytics-import) data into plausible
  - if someone (@Araq, @dom96?) gives me access to google analytics I can do that
- [ ] check in which other sites could we track (docs?)
  - having visibility on what is currently tracked in google analytics should help
  - https://github.com/nim-lang/Nim/pull/20454 added plausible in koch docs which now tracks non existent pages such as https://nim-lang.org/Nim/koch.html (we should find a way to fix that, I guess that real docs under nim-lang will be tracked once there is a new Nim release, correct?)
- [ ] add a link or section on the website to talk about the analytics (we could even show them as embedded, in that way we could track with analytics how many people look at them)
- [ ] I would like at some point (after points above are completed) to make a forum post/blog post to communicate the publicly available analytics to the broader community
  - in particular something which I have not explained yet is how I picked plausible over alternatives and what were the pros and cons (I got some question about this in telegram channel, but I would expect this is something that people would be curious about)
- [ ] later, if we are happy with plausible, we could remove google analytics

I would need help with the first two points above, the rest - with time - I should be able to manage.

**Comment by ringabout** [comment]
> (we should find a way to fix that, I guess that real docs under nim-lang will be tracked once there is a new Nim release, correct?)

True, it will be tracked by either 2.0 release or 1.6.10 release.

**Comment by pietroppeter** [comment]
thanks to support by @dom96 the import from google analytics has started. I will update when it is done!

> As to the non-existing doc link, either use different options for devel and stable documentation, or just let it be.

I think I will just let it be :) basically all links that start with `Nim/...` we know that they refer to devel docs.

**Comment by pietroppeter** [comment]
done!
<img width="1122" alt="image" src="https://user-images.githubusercontent.com/4997234/201528852-1fd0c837-ea11-45ad-8f23-cbc79777c4b7.png">
the G on the top right allows to disable view of GA data.

it appears only page views are reported without sources, details of pages and all that (I guess that's reasonable)
